config/21.1/aux.conf

```diff
@@ -1,3 +1,3 @@
 #ORIGIN						IGNORE
-lang/go						arm,arm64
+lang/go
```

Comment:
which go applications are not properly marked in ports.conf ? This doesn't change a lot since go will still be built in a ports run of needed and historically we tried to avoid go in any architecture that is not amd64. I'm not opposed either way, just that not a lot of go ports are enabled for arm64

Reply:
Yes, you are right, go will still be built in the ports building stage when needed.